RELEASES.md

```diff
@@ -1,3 +1,260 @@
+Version 1.13.0 (2016-11-10)
+===========================
+
+Language
+--------
+
+* [Stabilize the `?` operator][36995]. `?` is a simple way to propagate
+  errors, like the `try!` macro, described in [RFC 0243].
+* [Stabilize macros in type position][36014]. Described in [RFC 873].
+* [Stabilize attributes on statements][36995]. Described in [RFC 0016].
```

Comment:
Is this issue number correct? it has the same ID as the *Stabilize the `?` operator*, and redirects to the same PR.